src/renderer/components/dialogs/ViewGroup.tsx

```diff
@@ -238,6 +265,32 @@ function ViewGroupInner(props: {
                   {groupName} {chat.isProtected && <InlineVerifiedIcon />}
                 </p>
               </div>
+              <div className='group-separator'>
+                {tx('profile_shared_chats')}
+              </div>
+              <div style={{ marginLeft: '-20px' }}>
+                <ChatListPart
+                  isRowLoaded={isChatLoaded}
+                  loadMoreRows={loadChats}
+                  rowCount={chatListIds.length}
+                  width={400}
+                  height={CHATLISTITEM_CHAT_HEIGHT * chatListIds.length}
+                  itemKey={index => 'key' + chatListIds[index]}
+                  itemHeight={CHATLISTITEM_CHAT_HEIGHT}
+                >
+                  {({ index, style }) => {
```

Comment:
React newb question: How does passing functions to components work in react?

Reply:
Nvm. So this function here gets passed as `children` in `ChatListPart` which then passes that on to `FixedSizeList` https://web.dev/virtualize-long-lists-react-window/